provisioning/transport/http/src/Generated/RuntimeRegistration.cs

```diff
@@ -94,149 +93,133 @@ public RuntimeRegistration(DeviceProvisioningServiceRuntimeClient client)
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "idScope");
             }
+
             // Tracing
-            bool _shouldTrace = ServiceClientTracing.IsEnabled;
-            string _invocationId = null;
-            if (_shouldTrace)
+            bool shouldTrace = ServiceClientTracing.IsEnabled;
+            string invocationId = null;
+
+            if (shouldTrace)
             {
-                _invocationId = ServiceClientTracing.NextInvocationId.ToString(CultureInfo.InvariantCulture);
+                invocationId = ServiceClientTracing.NextInvocationId.ToString(CultureInfo.InvariantCulture);
                 var tracingParameters = new Dictionary<string, object>
                 {
                     { "registrationId", registrationId },
                     { "operationId", operationId },
                     { "idScope", idScope },
                     { "cancellationToken", cancellationToken }
                 };
-                ServiceClientTracing.Enter(_invocationId, this, "OperationStatusLookup", tracingParameters);
+                ServiceClientTracing.Enter(invocationId, this, "OperationStatusLookup", tracingParameters);
             }
+
             // Construct URL
-            string _baseUrl = Client.BaseUri.AbsoluteUri;
-            string _url = new Uri(
-                new Uri(_baseUrl + (_baseUrl.EndsWith("/", StringComparison.Ordinal) ? "" : "/")),
-                "{idScope}/registrations/{registrationId}/operations/{operationId}").ToString();
-
-#if NETSTANDARD2_1_OR_GREATER || NET5_0_OR_GREATER
-            _url = _url.Replace("{registrationId}", Uri.EscapeDataString(registrationId), StringComparison.Ordinal);
-            _url = _url.Replace("{operationId}", Uri.EscapeDataString(operationId), StringComparison.Ordinal);
-            _url = _url.Replace("{idScope}", Uri.EscapeDataString(idScope), StringComparison.Ordinal);
-#else
-            _url = _url.Replace("{registrationId}", Uri.EscapeDataString(registrationId));
-            _url = _url.Replace("{operationId}", Uri.EscapeDataString(operationId));
-            _url = _url.Replace("{idScope}", Uri.EscapeDataString(idScope));
-#endif
+            string baseUrl = Client.BaseUri.AbsoluteUri;
+            string url = new Uri(
+                new Uri(baseUrl + (baseUrl.EndsWith("/", StringComparison.Ordinal) ? "" : "/")),
+                $"{Uri.EscapeDataString(idScope)}/registrations/{Uri.EscapeDataString(registrationId)}/operations/{Uri.EscapeDataString(operationId)}").ToString();
```

Comment:
We can do this inline with string interpolation instead of string replacing. Odd original choice.